kratos/includes/properties.h

```diff
@@ -253,6 +258,29 @@ class Properties : public IndexedObject
         return rThisNode.GetValue(rV, SolutionStepIndex);
     }
 
+    /*
+    Custom GetValue in which we check the Accessor
+    */
+    template<class TVariableType, typename ...TArgs>
+    double GetValue(const TVariableType& rVariable, TArgs... Args) {
```

Comment:
This means that old GetValue still exists?, avoiding the penalty?

Reply:
In that case we should study using this: https://www.boost.org/doc/libs/1_81_0/libs/unordered/doc/html/unordered.html#compliance_open_addressing_containers_unordered_flat_set_unordered_flat_map

Reply:
The variadic templates are unnecessary in here if we are fixing the `AccessorType` signature.